src/server_manager/install_scripts/install_server.sh

```diff
@@ -168,17 +248,43 @@ function start_shadowbox() {
     -e "SB_METRICS_URL=${SB_METRICS_URL:-}"
     -e "SB_DEFAULT_SERVER_NAME=${SB_DEFAULT_SERVER_NAME:-}"
   )
-  docker run -d "${docker_shadowbox_flags[@]}" "${SB_IMAGE}" >/dev/null
+  local readonly RUN_DOCKER="docker run -d ${docker_shadowbox_flags[@]} ${SB_IMAGE}"
+  local readonly STDERR_OUTPUT
+  STDERR_OUTPUT=$(sg docker -c "$RUN_DOCKER 2>&1 >/dev/null")
```

Comment:
Let's create a function:
```
function safe_docker() {
  sg docker -c "$@"
}
```

Then the commands can use `safe_docker` and be more readable.

Reply:
Done in 47fc4bd.